src/test/resources/getTables/config.json

```diff
@@ -1,5 +1,6 @@
 {
   "parameters": {
-    "outputFile": "/code/tables.json"
+    "outputFile": "/code/tables.json",
+    "tables": []
```

Comment:
is this backwards compatible? Will config without it still work?

Reply:
doesn't need to be, cause this interfaces with the php script https://github.com/keboola/db-extractor-oracle/pull/76/files#diff-0470f12f4ee34ff6a4ee76ff8ac10b0bR59

Reply:
also it is https://github.com/keboola/java-oracle-exporter/pull/24/files#diff-c69eab5c9c99db4f5b1d03ba7654571aR52